conf/machine/include/rpi-base.inc

```diff
@@ -150,3 +150,20 @@ do_image_wic[recrdeps] = "do_build"
 # The kernel image is installed into the FAT32 boot partition and does not need
 # to also be installed into the rootfs.
 RDEPENDS:${KERNEL_PACKAGE_NAME}-base = ""
+
+RASPBERRYPI_3_FIRMWARE_PACKAGES += "\
```

Comment:
For easier maintenance, i would have split those variables in separate include files (one per device type)
and include them all in the rpi-base

Reply:
I will skip that for now as I don't want to introduce more confusion by further splitting the machine-specific confs/incs vs the confs that are applied for all machines.